### PR TITLE
117 displaying posts middle section grid

### PR DESCRIPTION
**Pull request by fatimaahmed14**

### CODE REVIEW

File: src/components/posts/PostForm.js [hunk 1]
```diff
@@ -3,19 +3,24 @@ import Button from '@mui/material/Button'
...
-        name="content"
-        onChange={handleChange}
-      />
-      <Button type="submit" variant="contained">
-        Submit
-      </Button>
-    </form>
+    <section className="new-post-form-section">
+      <div className="profile-picture-container">
+        <div className='"profile-picture'>FA</div>
```
Comment (lucianosimoni) [inline src/components/posts/PostForm.js:8]: Your could use the Initials API to return it dinamically
See wiki for more details on how to implement it
[How to use the Initials api](https://github.com/boolean-uk/team-dev-frontend-client/wiki/How-to-use-the-Initials-API)

File: src/components/posts/PostsPage.js [hunk 2]
```diff
@@ -3,6 +3,8 @@ import { useNavigate } from 'react-router-dom'
 import PostForm from './PostForm'
 import client from '../../utils/client'
 import './style.css'
+import TextField from '@mui/material/TextField'
+import Button from '@mui/material/Button'
```
Comment (lucianosimoni) [inline src/components/posts/PostsPage.js:7]: Should it be imported? Would be nice to confirm if this import is necessary on your code

File: src/components/posts/PostsPage.js [hunk 3]
```diff
@@ -54,13 +56,70 @@ const PostsPage = ({ loggedInUser }) => {
...
+                <form className="post-form">
+                  <TextField
+                    className="user-form-input"
+                    type="text"
+                    label="Add a comment..."
+                    variant="outlined"
+                    name="content"
+                    onChange={handleChange}
+                  />
+                  <Button type="submit" variant="contained">
```
Comment (lucianosimoni) [inline src/components/posts/PostsPage.js:110]: Related to the import.
This element should be a `<button>`, lowercase.

File: src/components/posts/PostForm.js (hunk 1 above)
Comment (MattVwaves) [reply src/components/posts/PostForm.js:8]: The dynamic initials will be done later - the main point of this issue was to get the styling right

File: src/components/posts/PostsPage.js (hunk 2 above)
Comment (MattVwaves) [reply src/components/posts/PostsPage.js:7]: When you go on the mui website this is how they suggest you use the components 
not sure the problem with this

File: src/components/posts/PostsPage.js (hunk 3 above)
Comment (MattVwaves) [reply src/components/posts/PostsPage.js:110]: Unsure as to what you mean here?

**Review by MattVwaves — APPROVED** [review]

**Pull request merged**